airbyte-db/db-lib/src/main/java/io/airbyte/db/jdbc/JdbcUtils.java

```diff
@@ -114,7 +115,13 @@ public static Map<String, String> parseJdbcParameters(final String jdbcPropertie
    * @return true: if ssl has not been set or it has been set with true, false: in all other cases
    */
   public static boolean useSsl(final JsonNode config) {
-    return !config.has(SSL_KEY) || config.get(SSL_KEY).asBoolean();
+    if (!config.has(SSL_KEY)) {
```

Comment:
Can we update the javadoc description to update what this method _now_ does so it's not de-synced from what this method originally did?

Reply:
> Can we update the javadoc description to update what this method _now_ does so it's not de-synced from what this method originally did?

updated javadoc